[2025-05-24T22:54:32.728+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-24T22:54:32.796+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: supplier_data_etl_pipeline.extract_sql_data manual__2025-05-24T22:54:15.604677+00:00 [queued]>
[2025-05-24T22:54:32.817+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: supplier_data_etl_pipeline.extract_sql_data manual__2025-05-24T22:54:15.604677+00:00 [queued]>
[2025-05-24T22:54:32.819+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 6
[2025-05-24T22:54:32.859+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): extract_sql_data> on 2025-05-24 22:54:15.604677+00:00
[2025-05-24T22:54:32.888+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=2172) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-24T22:54:32.906+0000] {standard_task_runner.py:63} INFO - Started process 2180 to run task
[2025-05-24T22:54:32.913+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'supplier_data_etl_pipeline', 'extract_sql_data', 'manual__2025-05-24T22:54:15.604677+00:00', '--job-id', '176', '--raw', '--subdir', 'DAGS_FOLDER/supplier_dag.py', '--cfg-path', '/tmp/tmpb7mdwgq1']
[2025-05-24T22:54:32.917+0000] {standard_task_runner.py:91} INFO - Job 176: Subtask extract_sql_data
[2025-05-24T22:54:33.233+0000] {task_command.py:426} INFO - Running <TaskInstance: supplier_data_etl_pipeline.extract_sql_data manual__2025-05-24T22:54:15.604677+00:00 [running]> on host a7d211f7b093
[2025-05-24T22:54:33.782+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='supplier_data_etl_pipeline' AIRFLOW_CTX_TASK_ID='extract_sql_data' AIRFLOW_CTX_EXECUTION_DATE='2025-05-24T22:54:15.604677+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-24T22:54:15.604677+00:00'
[2025-05-24T22:54:33.818+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-24T22:54:33.849+0000] {etl.py:77} INFO - connected to db source
[2025-05-24T22:54:33.919+0000] {etl.py:79} INFO - queried db source
[2025-05-24T22:54:33.920+0000] {etl.py:81} INFO - closed db connection
[2025-05-24T22:54:33.929+0000] {etl.py:83} INFO - db source pickled
[2025-05-24T22:54:33.932+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-05-24T22:54:33.936+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-05-24T22:54:34.018+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=supplier_data_etl_pipeline, task_id=extract_sql_data, execution_date=20250524T225415, start_date=20250524T225432, end_date=20250524T225434
[2025-05-24T22:54:34.084+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-05-24T22:54:34.240+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-24T22:54:34.248+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
